.pyup.yml

```diff
@@ -1,5 +1,6 @@
 schedule: "every day"
 search: False
+update: insecure
 requirements:
   - requirements.txt:
       updates: security
```

Comment:
> Do they really have two different spellings for the value depending on whether it's global or a specific requirements file? Do they really have two different sets of values?

The top-level option is [update](https://github.com/pyupio/pyup/blob/86a849b85fc27e60b3590b895ed8c15d0f8b7c09/pyup/config.py#L119-L126) and the per requirements file is also [update](https://github.com/pyupio/pyup/blob/86a849b85fc27e60b3590b895ed8c15d0f8b7c09/pyup/config.py#L61) though documented as update*s* with an s in the example: https://pyup.io/docs/configuration/

So we were using the update all default.

When I generated a config with security only updates against my fork pyup gave me a top level update: https://github.com/g-k/antenna/pull/1/files